src/Map/Map.test.js

```diff
@@ -25,22 +27,30 @@ describe('<Map />', () => {
     mount(<Map map={mockMap} onMapInit={onMapInit} />)
   })
 
-  it('should render with onMapInit callback', (done) => {
+  it('should render with onMapInit callback', async () => {
     // set the url with a view param
     window.history.replaceState(null, '', `${window.location.pathname}?view=49.618551,-97.280674,8.00,0.91`)
+    let testMap = null
     const onMapInit = map => {
+      // hoist map to closure for later expect
+      testMap = map
       // returned map should be an openlayers map
       expect(map).toBeInstanceOf(olMap)
       // do not respect the view location from the url since updateViewFromUrl={false}
       expect(map.getView().getCenter()).not.toEqual([-10829235.09370645, 6380475.798452517])
       expect(map.getView().getZoom()).not.toBe(8)
       // round off crazy long decimal
       expect(Number(map.getView().getRotation().toFixed(2))).not.toEqual(0.91)
-      done()
     }
 
     // using updateViewFromUrl={false} checks a separate if block inside Map's componentDidMount
-    mount(<Map onMapInit={onMapInit} updateViewFromUrl={false} />)
+    const { container } = render(<Map onMapInit={onMapInit} updateViewFromUrl={false} />)
+
+    // wait for async child render
+    await waitFor(() => {}, { container })
+
+    // make sure a <canvas> element is within the rendered html
+    expect(prettyDOM(testMap.getTargetElement())).toEqual(expect.stringContaining('<canvas'))
```

Comment:
the string match is only for `'<canvas'` since it renders with extra html props in the element: 
`<canvas someProps etc>`